acvm-repo/acir/src/proto/acir/circuit.proto

```diff
@@ -108,9 +108,9 @@ message BlackBoxFuncCall {
     Sha256Compression sha256_compression = 20;
   }
   message AES128Encrypt {
-    repeated FunctionInput inputs = 1;
-    repeated FunctionInput iv = 2;
-    repeated FunctionInput key = 3;
+    repeated ConstantOrWitnessEnum inputs = 1;
+    repeated ConstantOrWitnessEnum iv = 2;
+    repeated ConstantOrWitnessEnum key = 3;
```

Comment:
Just to note that this would be a breaking change even in Protobuf. Since we're not using it that's fine, but it's another reason not to bother with deprecating the other field. In fact I wouldn't even bother with `reserved` if we do this.